.github/actions/upload-sentry-sourcemaps/action.yml

```
@@ -91,28 +51,45 @@ runs:
           # Exit with the original exit code to preserve script success/failure status
           exit $original_exit_code
         }
-
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)